includes/wikia/services/WikiService.class.php

```diff
@@ -817,18 +817,29 @@ public function getDetails( Array $wikiIds = null ) {
 			$rows = $db->select(
 				array(
 					'city_visualization',
-					'city_list'
+					'city_list',
+					'city_verticals',
+					'city_domains',
+					'city_cat_mapping',
+					'city_cats'
 				),
 				array(
 					'city_list.city_id',
 					'city_list.city_title',
+					'city_list.city_dbname',
+					'city_list.city_description',
 					'city_list.city_url',
 					'city_list.city_lang',
-					'city_visualization.city_vertical',
+					'city_list.city_vertical',
 					'city_visualization.city_headline',
 					'city_visualization.city_description',
 					'city_visualization.city_main_image',
 					'city_visualization.city_flags',
+					'city_verticals.vertical_name',
+					'city_domains.city_domain',
+					'city_cat_mapping.cat_id',
+					'city_cat_mapping.city_id',
+					'city_cats.cat_name'
```

Comment:
Do we need to get all these columns? Not all of them seems to be used when fetching results in PHP.